app/views/conversations/new.mobile.haml

```diff
@@ -2,25 +2,26 @@
 -#   licensed under the Affero General Public License version 3 or later.  See
 -#   the COPYRIGHT file.
 
-:javascript
-  $(document).ready(function () {
```

Comment:
- Hash attribute should start with no space after the opening brace
- Hash attribute should end with no space before the closing brace